daemon/graphdriver/fsdiff.go

```diff
@@ -44,6 +44,7 @@ func NewNaiveDiffDriver(driver ProtoDriver, uidMaps, gidMaps []idtools.IDMap) Dr
 // Diff produces an archive of the changes between the specified
 // layer and its parent layer which may be "".
 func (gdw *NaiveDiffDriver) Diff(id, parent string) (arch archive.Archive, err error) {
+	startTime := time.Now()
```

Comment:
I'd propose to use `time.Now().Truncate(time.Second)` here and `startTime.Add(time.Second).Sub(time.Now())` in sleep. It looks more understandable to me.